### PR TITLE
Refactor: Cleans up scss

### PR DESCRIPTION
**Pull request by asbjornh**
The scss files contained some bloat that probably isn't necessary.

I left some things in `site.css`:
- `box-sizing` because I can't imagine a project where having `content-box` or `padding-box` as default makes sense.
- `-webkit-font-smoothing`: This is necessary in order for fonts to render properly on MacOS. I don't think everyone cares about this (except for the designers of course) so I think it's a good idea to have it as a default.
- `.mouse-user`: Half the point of including `input-detection.js` is to disable focus outlines for mouse users

### CODE REVIEW

File: static-files/source/scss/style.scss [hunk 1]
```diff
@@ -1,7 +1,6 @@
 @import "fonts";
 @import "vars";
 @import "mixins";
```
Comment (pellebjerkestrand) [inline static-files/source/scss/style.scss]: The mixins file doesn't exist anymore, does it?

Comment (pellebjerkestrand) [reply static-files/source/scss/style.scss]: Or rather: there doesn't seem to be anything in it anymore, so it should be removed.

Comment (asbjornh) [reply static-files/source/scss/style.scss]: Both `vars.scss` and `mixins.scss` still exist as empty files. My thinking is that I (and probably others too) want those files for every new project and so I'd like to save the keystrokes and mouse clicks that it takes to create and import those files. To me much of the appeal of a tool like this is to be relieved of tedious manual labour

**Comment by asbjornh** [comment]
Bump! @pellebjerkestrand 

Merging this tonight unless there's any more activity

**Pull request merged**